src/BikeDataProject.API/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Get .NET Core v3.1 build image
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1 AS build
+
+# copy api assemblies and files
+RUN mkdir /var/app
+RUN mkdir /var/app/logs
+COPY ./bin/release/netcoreapp3.1/publish /var/app
+# COPY appsettings.json /var/app/appsettings.json
+
+# couple data folder data volume.
+VOLUME ["/var/app/data"]
+VOLUME ["/var/app/logs"]
+
+# SET base url
+ENV ASPNETCORE_URLS http://+:5000
```

Comment:
Should the API run on `:5000` instead of `:80`?

Reply:
This is mapped to an external port by docker later when in production so it doesn't matter really. We can go back to 80 if you think that's clearer :+1: